lib/main.test.ts

```
@@ -630,3 +636,60 @@ describe("secureFetch", () => {
     global.kinde.secureFetch = backup;
   });
 });
+
+describe("denySelection", () => {
+  it("should return the results from kinde environment", async () => {
+    const backup = global.kinde.plan;
+    delete global.kinde.plan;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)